### PR TITLE
B/attrupdate

### PR DESCRIPTION
**Pull request by vitiral**
there was an error when setting items like "update" -- which are attributes, but the setitem getitem should go around that fact.

The error was slightly sloppy coding. The `actual` value is now used, which did things correctly in the first place.

### CODE REVIEW

File: tests/test_properties.py [hunk 1]
```diff
@@ -38,6 +38,14 @@ def test_descriptor(self):
         attr.enum = 0
         assert attr.enum == 'a'
 
+    def test_set_update(self):
+        '''Makes sure that object can set values like "update" that are
```
Comment (cloudformdesign) [inline tests/test_properties.py]: Note: this test worked immediately without any changes necessary.

File: tests/test_properties.py [hunk 2]
```diff
@@ -62,6 +70,22 @@ def test_error(self):
         convert = TypedDict(mydict)
         self.assertRaises(ValueError, setattr, convert, 'c', 'hello')
 
+    def test_set_update(self):
+        '''Makes sure that object can set values like "update" that are
```
Comment (cloudformdesign) [inline tests/test_properties.py]: This test failed at first.

File: tests/test_properties.py [hunk 3]
```diff
@@ -62,6 +70,22 @@ def test_error(self):
         convert = TypedDict(mydict)
         self.assertRaises(ValueError, setattr, convert, 'c', 'hello')
 
+    def test_set_update(self):
```
Comment (hansmosh) [inline tests/test_properties.py]: Two tests by the same name? I have a feeling one of these isn't running.

Comment (cloudformdesign) [inline tests/test_properties.py]: they are in different test classes

Comment (hansmosh) [inline tests/test_properties.py]: Ah, I see! Same file, different class. It's hard to see that on GitHub.

**Pull request merged**